### PR TITLE
Disable enemy-intercept and automatic code blue, by request of Necaladun.

### PR DESCRIPTION
**Pull request by tigercat2000**
This commit does the following:
 - Disables "Enemy Intercept" entirely.
 - Disables automatic blue alert.

@Necaladun

### CODE REVIEW

**Comment by SirBloodBuske** [comment]
yesssss finnallyyy

**Comment by Earthdivine** [comment]
Rejoice! It's the return of the door!

**Comment by Robertha89** [comment]
Now you have to break in to git dem delicious weapons, or just order 7 crates from cargo.

**Comment by SamCroswell** [comment]
:-1: on all changes.

The door is a horrible decision. Period. There are items in there that security may need at a moment's notice to deal with situations that may not warrant code red, or for situations where taking the time to call for code red might be the difference between dead nukeops and a radioactive debris field where the Cyberiad uses to be. On top of that -- There are a myriad of ways people can cheese the system using science or ventcrawling that security cannot readily replicate, meaning security is one of the most poorly equipped departments to get inside _its own armory_. This is a bad idea. Period.

**Comment by tigercat2000** [comment]
@SamCroswell Talk to Necaladun.

**Comment by SamCroswell** [comment]
@tigercat2000 
He ought to be watching this thread, no?

**Comment by tigercat2000** [comment]
@SamCroswell True.

**Comment by Spacemanspark** [comment]
:+1:

**Comment by Robertha89** [comment]
:+1: for this, and all of the issues dave mentioned can be fixed by having two armories, as it should be, but that can be fixed in a seperate pr

**Comment by FalseIncarnate** [comment]
:+1: to the removal of the intercept message
:+1: to the removal of automatic code blue
:+1: & :-1: to the door.

I like the idea of limiting some of the more harmful weapons to code red and up (would help cut down on shitcurity validhunters), but Dave also made a good point about the access to these things when needed. Would it be possible to lock some equipment behind a shutter so that it can only be snatched on code red? Or give the HoS/warden an override button for the armory so that in the event of nuke ops storming the station, they can easily unlock the heavy weapons if code red can't be called for some reason.

**Comment by Spacemanspark** [comment]
Technically there are too armories, but the second one (Gamma) is literally never fucking used.

**Comment by Fox-McCloud** [comment]
This is beyond a terrrible idea, especially the armory.

We had two armories, before---and it didn't work well for a lot of scenarios. If there ended up being spiders, carp, or some other low to moderate level threat, then security couldn't do anything to handle it unless the HoS, HoP, or Captain specifically hopped on it themselves.

This also means nukies will walk all over the entirety of the security force unless the heads of staff call code red immediately; sec will literally only have non-lethal options to deal with them; if they happen to kill the HoS or Cap together---then code red isn't going to get called and the crew is going to be literally toothless to fight against them. This also means that Nuke ops will raid the armory far more often than they currently do--why? They can break in and steal all of its contents, brazenly and not have to worry about repercussion because sec won't have anything by the time they're done. 

This also means a return to sneaking into and looting the entirety of the armory by breaching it or other means as there's no way to keep an eye on it and no way of stopping people from doing so--telescientist wants to loot the entire armory then space all the gun (or keep them for himself)? You literally have to call code red JUST to stop him. Not a good thing..same goes for chemists who make thermite or engineers who break in from the back. 

When this change went through, everyone was extremely skeptical, myself included, over the consequences of the armory being freely available. I was quite surprised; the armory almost never gets looted day to day---once in a while you do get some warden or HoS who literally takes everything and roams the halls with the gear---but generally speaking it's hardly ever and typically utilized only when it's absolutely needed.

**Comment by SamCroswell** [comment]
@FalseIncarnate 
The issue with that is the only people who have access to the armory currently are the HoS and Warden. All other officers have to be let in, so under that system they'd be able to access it under the exact same circumstances: When the HoS or Warden deems it appropriate.

This is a solution to a problem that does not exist, and it _causes_ problems to boot. As I said earlier: Security rarely, if ever, abuses the gear in the armory. Infact, only a small number of the items in the armory are even lethal.

The items in the armory are as follows:
Energy Guns - Generally used when one wants to take someone in alive, but lethal force might become neccisary
Laser Guns - Generally used when lethal force is the only option.
Advanced Tasers - Standard security gear for nonlethal takedowns, put in the armory to prevent security from carrying more than one at a time. It's important to note that if we restrict armory access, officers hired onto the force by the HoP will be without tasers entirely.
RIG suits - For use in the event that security must go into an underpressurized area.
Armor (Riot, Bulletproof, Ablative) - Specialized armor protecting security from the 3 main sources of potential combat damage depending on the kind of armor worn.
Ion Rifle - Used for stopping rogue synthetics or mechs.
Riot Shotgun - Loaded with beanbags by default, but there is also a box of shotgun darts loaded with haloperidol. Extremely useful for when you have a meth'd up murderer you cannot stun, as haloperidol purges such drugs from the person.

Of the items listed, I would say that in the situations they are used under very few of them actually warrant code red to access them. Why security should have to get code red called to stop one otherwise very stun-resistant methed up Syndie with a riot shotgun is beyond me, nor why they should have to call code red to get a RIG suit to follow a rogue engineer into a depressurized room. As for the Ion rifle - A rogue AI might warrant code red, but by the time the command staff organize to call code red the AI is almost certainly going to be near victory, and that's assuming the AI hasn't simply bolted every head's office down already, or simply cut power to the armory APC so that the bolts don't raise when code red is called.Similarly to above, a single rogue Odysseus probably doesn't warrant a code red.

Regarding advanced tasers: Putting them in the armory was a very wise move. Officers are now restricted to carrying one taser with limited charge, for the most part. Under this system we would either have to put tasers back in security's locker, opening the door to officers carrying backpacks full of tasers, or we have to call code red every time the HoP brings on a new officer to give him a taser.

And *all of this* assumes command is competent, will listen to the HoS when he says there's a serious threat, and that they have time to swipe in a crisis. Previously, when this system was around, I saw legions of officers charging ops with batons because they had no other choice.

In short: The code red door was removed for a very good reason, and I very strongly believe it should stay removed for these reasons.

**Comment by fj45** [comment]
Code red armoury is a nice idea but it just doesn't work. Access is restricted enough to the armoury that it isn't an issue.

Remove the automatic code blue and intercept though, never got the point of them.

**Comment by Iamgoofball** [comment]
why not just make the intercept and auto code blue a config option???

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -272,8 +272,9 @@ Implants;
 	return 0
 
 
-/datum/game_mode/proc/send_intercept()
-	var/intercepttext = "<FONT size = 3><B>Cent. Com. Update</B> Requested status information:</FONT><HR>"
+/datum/game_mode/proc/send_intercept() //Disabled.
+	return
+/*	var/intercepttext = "<FONT size = 3><B>Cent. Com. Update</B> Requested status information:</FONT><HR>"
 	intercepttext += "<B> In case you have misplaced your copy, attached is a list of personnel whom reliable sources&trade; suspect may be affiliated with the Syndicate:</B><br>"
```
Comment (Iamgoofball) [inline code/game/gamemodes/game_mode.dm]: aren't bonus TC handed out with people on the intercept report if they are actually antags

should remember to config option that part of it too when you make this a config option

Comment (Fox-McCloud) [inline code/game/gamemodes/game_mode.dm]: Only on Bay--not here or TG.

Comment (Spacemanspark) [inline code/game/gamemodes/game_mode.dm]: That used to be the case here, why isn't it still?

Comment (Iamgoofball) [inline code/game/gamemodes/game_mode.dm]: because rng "oh now you have extra TCs" can fuck with the balance of costs on some shit

some things are balanced around "you can only buy 1"

Comment (TheDZD) [inline code/game/gamemodes/game_mode.dm]: Also because it encouraged people to set their character's "Nanotrasen Relationship" to "opposed", as it would increase their chance of being on the report and getting free TCs.

Comment (Iamgoofball) [inline code/game/gamemodes/game_mode.dm]: wait WHAT

so that explains why I kept ending up on the report on BS12

Comment (ZomgPonies) [inline code/game/gamemodes/game_mode.dm]: The balance was: extra TC when you roll traitor, but you might get falsely accused if you're not.

**Comment by Necaladun** [comment]
A lot of good points on the code-red armoury problems here, the main intent was to have an actual reason to go to red alert. Currently it has basically no effect. 

Making the armoury change a separate thing to discuss would be best - there are a few options there, such as making a green/blue alert section, with extra tazers, hardsuits, etc and a red alert section with grenade launchers and whatnot.

**Comment by adrkiller59** [comment]
Just want to point out that THERE ARE two armories currently. The room with flashers and flashbang/handcuffs boxes is an armory, it just has little equipment, and only the warden/HoS/Cap'n can access it. If i remember correctly that is..

**Comment by tigercat2000** [comment]
The armoury change has been reverted.

**Comment by ZomgPonies** [comment]
:+1:  in this case.

**Comment by SamCroswell** [comment]
Then, while I still disagree with the rest of the PR, I'm gonna not be a massive bitchlord and hold up the PR even longer by debating that as well.

:+1:

**Comment by TheDZD** [comment]
:+1: now that the armory door change is removed.

**Comment by Regen1** [comment]
:+1:

**Comment by Plotron** [comment]
Good that the armory thing has been figured out (I agreed with Fox and Sam).
I couldn't care less about the intercept and blue alert. It's silly, loud and obnoxious.
Gives more incentive to roleplay alert levels.
